packages/manager/apps/telecom/src/app/telecom/pack/migration/meeting/pack-migration-meeting.html

```diff
@@ -0,0 +1,135 @@
+    data-ng-if="$ctrl.showMeetingSlots && !$ctrl.loading"
+>
+    <div class="row">
+        <p
+            class="font-weight-bold"
+            data-translate="telecom_pack_migration_meeting_description"
+        ></p>
+
+        <div
+            data-ng-show="$ctrl.meetingSlots.canBookFakeMeeting && $ctrl.meetingSlots.slots.length"
+        >
+            <div class="input-group mt-3 col-md-12">
+                <input
+                    type="radio"
+                    id="radioFakeMeeting"
+                    data-ng-model="$ctrl.selectedMeeting.meetingSlots.fakeMeeting"
+                    data-ng-disabled="!$ctrl.meetingSlots.canBookFakeMeeting"
+                    data-ng-value="true"
+                />
+                <label
+                    for="radioFakeMeeting"
+                    class="pl-2"
+                    data-translate="telecom_pack_migration_fake_meeting"
+                >
+                </label>
+            </div>
+            <div class="input-group mt-3 col-md-12">
+                <input
+                    type="radio"
+                    id="radioNoFakeMeeting"
+                    data-ng-model="$ctrl.selectedMeeting.meetingSlots.fakeMeeting"
+                    data-ng-value="false"
+                />
+                <label
+                    for="radioNoFakeMeeting"
+                    class="pl-2"
+                    data-translate="telecom_pack_migration_no_fake_meeting"
+                >
+                </label>
+            </div>
+        </div>
+        <p
+            class="alert alert-info"
+            role="alert"
+            data-ng-if="$ctrl.meetingSlots.canBookFakeMeeting && !$ctrl.meetingSlots.meetingSlots.length"
+            data-translate="telecom_pack_migration_no_erdv_info"
+        ></p>
+
+        <p
+            class="mt-3"
+            data-translate="telecom_pack_migration_contact_name_info"
+        ></p>
+
+        <div class="row">
+            <div class="form-group col-md-6">
+                <label
+                    class="control-label required"
+                    for="contactNameId"
+                    data-translate="telecom_pack_migration_contact_name"
+                ></label>
+                <input
+                    id="contactNameId"
+                    class="form-control"
+                    required
+                    pattern=".+"
+                    data-ng-model="$ctrl.contactName"
+                    placeholder="{{'telecom_pack_migration_contact_name'| translate}}"
+                />
+            </div>
+        </div>
+        <div class="row">
+            <div
+                class="col-md-6 mt-2 font-italic font-weight-lighter"
+                data-ng-show="$ctrl.meetingSlots"
+                data-translate="telecom_pack_migration_mandatory_field"
+            ></div>
+        </div>
+    </div>
+
+    <p
+        class="mt-3"
+        data-ng-show="!$ctrl.selectedMeeting.meetingSlots.slot"
```

Comment:
if duable we avoid the `data-ng-show` directive.
Prefer to use `data-ng-if` directive